ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/model/response/EthSignerResponse.java

```diff
@@ -41,6 +45,18 @@ public EthSignerResponse(
     this.body = body;
     this.headers = headers;
     this.status = status;
+    this.statusLine = Optional.empty();
+  }
+
+  public EthSignerResponse(
+      final Iterable<Entry<String, String>> headers,
+      final String body,
+      final HttpResponseStatus status,
+      Optional<String> statusLine) {
```

Comment:
Be nicer to have the constructor arg as String instead of Optional<String> as we already have the alternative constructor creating an Optional.empty

Reply:
done.